### ISSUE TITLE
Option for no Maven or Gradle

### ISSUE THREAD

**Issue by Max604** [DemonWav/MinecraftDevIntelliJ#309]
Sometimes I just want to create a quick simple project without maven or gradle.
Do you think you could add this feature?

Thanks.

**Comment by kinggoesgaming** [comment]
Problem is that various API dependence still need to be downloaded and versions to be maintained which is quite a burden for a project like this

**Comment by Max604** [comment]
Is there an easy way I can skip that step by editing the source?

**Comment by Mumfrey** [comment]
In your mind, if you don't use a build system, what actually manages the project?

If you expect the plugin to duplicate everything that build system would do then that's frankly insane, since why would the plugin completely reinvent what the build system already does perfectly. And if you *don't* expect the plugin to duplicate that functionality, then what you're asking for is a plain Java project, which you can create already.

How does this fantasy "project with no build system" actually *work*?

Basically what you've done is go to MacDonalds and said: "hey, I'd like a cheeseburger with no cheese, or burger". You just a get a bun, and frankly you can get a bun from your nearest convenience store.

**Comment by DenWav** [comment]
Nope.
